### ISSUE TITLE
Fix Broken Social Media Links in Footer Icons by adding appropriate alt tag and ensure accessibility

### ISSUE THREAD

**Issue by anythingkartik** [piyushg-07/blogzen-OpenSource#91]
The social media icons in the footer are missing alt attributes, which are essential for accessibility. alt attributes provide descriptive text for screen readers, improving accessibility for users with visual impairments or those who have images disabled in their browsers. This issue should be addressed by adding appropriate alt attributes to the <img> tags of the social media icons in the footer. Enhancing accessibility in this manner will ensure that all users can access and understand the content of the website.

File being discussed- index.html

**Comment by anythingkartik** [comment]
Please assign me this task

**Comment by Aditi-Gopinath** [comment]
Kindly assign me this issue under GSSoC'24
I would like to solve this issue.
